### PR TITLE
Updated Header PO

### PR DESCRIPTION
**Pull request by rsavu98**

### CODE REVIEW

File: src/main/java/com/ss/greencity/locators/HeaderLocators.java [hunk 1]
```diff
@@ -4,16 +4,17 @@
...
     HEADER(By.className("header")),
-    LOGO(By.cssSelector("")), //implement if necessary
-    NEWS(By.cssSelector("")), //implement if necessary
-    TIPS(By.cssSelector(".title-list.word-wrap p")), //implement if necessary
-    PLACES(By.cssSelector("")), //implement if necessary
-    ABOUT(By.cssSelector("")), //implement if necessary
-    HABITS(By.cssSelector("")), //implement if necessary
-    SEARCH(By.cssSelector("img[alt='search']")), //search button
+    LOGO(By.cssSelector("img[alt='logo']")),
+    NEWS(By.xpath("//div[@class='navigation-menu-left']/ul/li[1]")),
```
Comment (ilonaGrigorak) [inline src/main/java/com/ss/greencity/locators/HeaderLocators.java]: NEWS(By.cssSelector((".navigation-menu-left a[href='#/news']"))),

File: src/main/java/com/ss/greencity/locators/HeaderLocators.java [hunk 2]
```diff
@@ -4,16 +4,17 @@
...
+    TIPS(By.xpath("//div[@class='navigation-menu-left']/ul/li[2]")),
```
Comment (ilonaGrigorak) [inline src/main/java/com/ss/greencity/locators/HeaderLocators.java]: TIPS(By.cssSelector((".navigation-menu-left a[href='#/tips']")))

File: src/main/java/com/ss/greencity/locators/HeaderLocators.java [hunk 3]
```diff
@@ -4,16 +4,17 @@
...
+    PLACES(By.xpath("//div[@class='navigation-menu-left']/ul/li[3]")),
```
Comment (ilonaGrigorak) [inline src/main/java/com/ss/greencity/locators/HeaderLocators.java]: PLACES(By.cssSelector((".navigation-menu-left a[href='#/map']")))

File: src/main/java/com/ss/greencity/locators/HeaderLocators.java [hunk 4]
```diff
@@ -4,16 +4,17 @@
...
+    ABOUT(By.xpath("//div[@class='navigation-menu-left']/ul/li[4]")),
```
Comment (ilonaGrigorak) [inline src/main/java/com/ss/greencity/locators/HeaderLocators.java]: ABOUT(By.cssSelector((".navigation-menu-left a[href='#/about']"))),

File: src/main/java/com/ss/greencity/locators/HeaderLocators.java [hunk 5]
```diff
@@ -4,16 +4,17 @@
...
+    HABITS(By.xpath("//div[@class='navigation-menu-left']/ul/li[5]")),
```
Comment (ilonaGrigorak) [inline src/main/java/com/ss/greencity/locators/HeaderLocators.java]: HABITS(By.cssSelector((".navigation-menu-left a[href*='#/profile']")))

**Review by ilonaGrigorak — CHANGES_REQUESTED** [review]
review of header updates by Ruslana

File: src/main/java/com/ss/greencity/locators/HeaderLocators.java (hunk 1 above)
Comment (rsavu98) [reply src/main/java/com/ss/greencity/locators/HeaderLocators.java]: @ilonaGrigorak, thanks, updated.

**Pull request merged**